diff --git a/.cargo/config.toml b/.cargo/config.toml
index bc7bfd6..6e4022f 100644
--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -7,3 +7,6 @@ dev = "run --package uv-dev --features dev"
 # See: https://github.com/astral-sh/ruff/issues/11503
 [target.'cfg(all(target_env = "msvc", target_os = "windows"))']
 rustflags = ["-C", "target-feature=+crt-static"]
+
+[target.loongarch64-unknown-linux-musl]
+rustflags = ["-C", "target-feature=+crt-static"]
diff --git a/crates/uv-python/fetch-download-metadata.py b/crates/uv-python/fetch-download-metadata.py
index 9bf88e7..7c04aa4 100755
--- a/crates/uv-python/fetch-download-metadata.py
+++ b/crates/uv-python/fetch-download-metadata.py
@@ -191,7 +191,7 @@ class CPythonFinder(Finder):
     implementation = ImplementationName.CPYTHON
 
     RELEASE_URL = (
-        "https://api.github.com/repos/astral-sh/python-build-standalone/releases"
+        "https://api.github.com/repos/loong64/python-build-standalone/releases"
     )
 
     FLAVOR_PREFERENCES = [
diff --git a/dist-workspace.toml b/dist-workspace.toml
index 7280655..137c402 100644
--- a/dist-workspace.toml
+++ b/dist-workspace.toml
@@ -36,7 +36,9 @@ targets = [
     "x86_64-pc-windows-msvc",
     "i686-unknown-linux-gnu",
     "i686-unknown-linux-musl",
-    "i686-pc-windows-msvc"
+    "i686-pc-windows-msvc",
+    "loongarch64-unknown-linux-gnu",
+    "loongarch64-unknown-linux-musl"
 ]
 # Whether to auto-include files like READMEs, LICENSEs, and CHANGELOGs (default true)
 auto-includes = false
