diff --git a/crates/uv-python/fetch-download-metadata.py b/crates/uv-python/fetch-download-metadata.py
index 71dd56a..04ccb47 100755
--- a/crates/uv-python/fetch-download-metadata.py
+++ b/crates/uv-python/fetch-download-metadata.py
@@ -176,7 +176,7 @@ class CPythonFinder(Finder):
     implementation = ImplementationName.CPYTHON
 
     RELEASE_URL = (
-        "https://api.github.com/repos/astral-sh/python-build-standalone/releases"
+        "https://api.github.com/repos/loong64/python-build-standalone/releases"
     )
 
     FLAVOR_PREFERENCES = [
diff --git a/dist-workspace.toml b/dist-workspace.toml
index 60e3001..f352ceb 100644
--- a/dist-workspace.toml
+++ b/dist-workspace.toml
@@ -34,7 +34,9 @@ targets = [
     "x86_64-pc-windows-msvc",
     "i686-unknown-linux-gnu",
     "i686-unknown-linux-musl",
-    "i686-pc-windows-msvc"
+    "i686-pc-windows-msvc",
+    "loongarch64-unknown-linux-gnu",
+    "loongarch64-unknown-linux-musl"
 ]
 # Whether to auto-include files like READMEs, LICENSEs, and CHANGELOGs (default true)
 auto-includes = false
